### DISCUSSION TITLE
Can I store the values as an array of values instead of an array of objects

### DISCUSSION THREAD

**Discussion by barrymichaeldoyle** [csandman/chakra-react-select#264] [Q&A]
TLDR;

I'm using this with `react-hook-form`, but my validation schema checks for an array of strings for my select value. Instead the values coming from `onChange` and stored on my Select component come in the format `[{ label: "Value 1", value: "value1"}, ...]`. I'm expecting `['value1', ...]` to be evaluated, not the array of objects.

I've posted a StackOverflow question here:

https://stackoverflow.com/questions/76383509/react-hook-form-with-react-select-how-to-save-values-as-an-array-of-values-inste

I know this is technically a `react-select` issue but this library has much better support than `react-select` does so I'm trying my luck here as well.

**Comment by csandman** [comment]
Sure, there are a couple different issues/discussions outlining how to do this already, but here's an example: https://codesandbox.io/s/qf1v1r?file=/app.tsx

Here's some discussions with similar requests/solutions: #105 #111 #175 #251
